### ISSUE TITLE
Next Step Microgrant: KopperDAO

### ISSUE THREAD

**Issue by Islandersfan2016** [filecoin-project/devgrants#329]
### 1. What is your project? (max 100 words)

Kopper allows content creators to form a close community with their fans by putting exclusive content into IPFS files. The community of fans can vote, give tips, and watch live content from the content creators. The community of fans can vote, give tips, and watch live content from the content creators. Each content creator is their own DAO.
This project utilizes Textile’s bridge Filecoin/Polygon (low gas fees). Also a Chainlink external adapter that connects the IPFS file to smart contract holding fan addresses. 

   
 Coming Soon!

### 2. How are you planning to improve this project? (max 200 words)

We plan on expanding the project to hold more profiles and launch the platform. We need funding for extra developers to join the team. Currently, there is only myself as the sole developer with 1 Domain Expert. 
 
### 3. Will the work be Open Source?
 Yes, this will be Open Source

### 4. If selected, do you agree to share weekly updates and a grant report upon conclusion?
 Yes we will give weekly updates.
  
### 5. Does your proposal comply with our Community Code of Conduct?
 Yes it does.

### 6. Links and submissions
Yes
  
* Have you submitted to the Filecoin Community Showcase ([instructions](https://github.com/filecoin-project/community#showcase-your-project))?
   https://github.com/filecoin-project/community/discussions/305

* Have you created a thread introducing your project in the community discussion board's [ Project Showcase category](https://github.com/filecoin-project/community/discussions/categories/project-showcase)?
  
  https://github.com/filecoin-project/community/discussions/305

* If your project began at a hackathon, have you submitted it for the relevant Protocol Labs prizes? Include links here if available:
  
### Additional questions:
* For each team member(s), please list name, email, Github account, and role in the project. 
Lamont Veal
Marcel Myers

* If your project was created as part of an event or hackathon:
  * What was the name of the event? (e.g. ETHGlobal NFTHack, Cal Hacks hello:world, Chainlink, CivHacks, GameDevJ, ETHGlobal Scaling Ethereum) EthGlobal HackFS
  * Please link to your hackathon submission, proving you're a team that submitted for a Protocol Labs prize.

**Comment by SeedingTrees** [comment]
This microgrant has been approved! Look out for an email to submit payment details. Let's use your existing Community Discussion thread for updates.

**Comment by ErinOCon** [comment]
Hi @Islandersfan2016, thank you for your [recent proposal](https://github.com/filecoin-project/devgrants/issues/1308). Before proceeding, can you provide a link to the building progress made for this project and describe how the microgrant was applied?

**Comment by Islandersfan2016** [comment]
Hi Erin,

Yes, we have a site and demo for Muse Filez. The project and concept has
changed quite a bit over the past year and a half. I will attach some links
right now. It is no longer called KopperDAO.

We have created a Social media platform where content creators can sell
tickets to fans via the Unlock Protocol (NFTs) exclusive content within
their Web3.Storage accounts. Also, back in October 2022 we applied again
for the Open Grant (marketing, legal fees, security audit) to launch the
DAPP to the public. You recommended that we enter into a Hackathon again.
So we entered into the FVM Space Warp Hackathon.

It looks like the confusion of which project this was, is the reason for us
entering another Hackathon. I should have communicated this better to you,
I apologize.

The Micro Grant of $5,000 was divided between myself and the co-founder
Pete Myers for basic living expenses as we continued to build out the DAPP
over a 3 month span.

I have attached the Github, DAPP, and a demo video. This DAPP can be
launched to the general public once we have an audit, have hired a lawyer
and helped with marketing.

MuseFilez (github.com) <https://github.com/MuseFilez>

Muse App (fleek.co) <https://divine-rain-1381.on.fleek.co/>

 Muse App — Mozilla Firefox 2023-02-08 18-30-58.mp4
<https://drive.google.com/file/d/1OO9Afxk2PZqrQ3S393DZpoomFJsqY2vL/view?usp=drive_web>

Thank you,
Lamont V.
